zmon_agent/discovery/kubernetes/cluster.py

```diff
@@ -720,15 +770,15 @@ def get_postgresql_clusters(kube_client, cluster_id, alias, environment, region,
             'shards': {
                 'postgres': '{}:{}/postgres'.format(service_dns_name, POSTGRESQL_DEFAULT_PORT)
             },
-            'expected_replica_count': ss.get('replicas', 0),
+            'expected_replica_count': pg.get('expected_instance_count', 0),
             'current_replica_count': ss.get('actual_replicas', 0),
             'statefulset_error': statefulset_error,
             'deeplink1': '{}/#/status/{}'.format(hosted_zone.format('pgui', alias), version),
             'icon1': 'fa-server',
             'deeplink2': '{}/#/clusters/{}'.format(hosted_zone.format('pgview', alias), version),
             'icon2': 'fa-line-chart',
-            'uid': ss.get('metadata', {}).get('uid', ''),
-            'namespace': ss.get('metadata', {}).get('namespace', '')
+            'uid': pg.get('uid', ''),
+            'namespace': pg.get('namespace', '')
```

Comment:
this should not change imho, while true we expect it to map 1:1 but lets not assume it for the future.